.prettierrc

```diff
@@ -1,10 +1,11 @@
-useTabs: true
-tabWidth: 4
-printWidth: 80
-singleQuote: true
-trailingComma: es5
-bracketSpacing: true
-parenSpacing: true
-jsxBracketSameLine: false
-semi: true
-arrowParens: always
+{
+	"useTabs": true,
+	"printWidth": 80,
```

Comment:
In VSCode it no longer seems happy with yaml for the prettierrc. I've changed it to JSON, which fixes that.